sklearn/feature_extraction/image.py

```diff
@@ -339,25 +339,45 @@ def extract_patches_2d(image, patch_size, max_patches=None, random_state=None):
     Examples
     --------
 
-    >>> from sklearn.feature_extraction import image
-    >>> one_image = np.arange(16).reshape((4, 4))
-    >>> one_image
-    array([[ 0,  1,  2,  3],
-           [ 4,  5,  6,  7],
-           [ 8,  9, 10, 11],
-           [12, 13, 14, 15]])
-    >>> patches = image.extract_patches_2d(one_image, (2, 2))
-    >>> print(patches.shape)
-    (9, 2, 2)
-    >>> patches[0]
-    array([[0, 1],
-           [4, 5]])
-    >>> patches[1]
-    array([[1, 2],
-           [5, 6]])
-    >>> patches[8]
-    array([[10, 11],
-           [14, 15]])
+        from sklearn.datasets import load_sample_images
+        from sklearn.feature_extraction import image
+
+        # Use the array data from the first image in this dataset:
+        one_image = load_sample_images().images[0]
+        print(f'Image shape: {one_image.shape}')
```

Comment:
This string formatting is only available in python 3.6. We are still supporting python 3.5. Use normal string with format here

Reply:
OK.